grafana-cloud-integration.md

```diff
@@ -7,72 +7,116 @@ This document will show you how to integrate the Grafana Cloud Prometheus metric
-      wal_directory: /tmp/grafana-agent-wal
+      wal_directory: /var/lib/agent/wal
       global:
         scrape_interval: 60s
         external_labels:
-          cluster: cloud
+          cluster: <REPLACE-WITH-YOUR-CLUSTER-NAME>
       configs:
       - name: integrations
         remote_write:
-        - url: <grafana_prometheus_remoteWrite_url>
+        - url: https://<REPLACE-WITH-GRAFANA-PROM-REMOTE-WRITE-ENDPOINT>
           basic_auth:
-            username: # API key name
-            password: # API key password
-        scrape_configs:
-        - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
-          job_name: integrations/kubernetes/cadvisor
+            username: <REPLACE-WITH-GRAFANA-PROM-REMOTE-WRITE-USERNAME>
+            password: <REPLACE-WITH-GRAFANA-PROM-REMOTE-WRITE-API-KEY>
+        scrape_configs: #Need further scrape config update
+        - job_name: kubecost
+          honor_labels: true
+          scrape_interval: 1m
+          scrape_timeout: 10s
+          metrics_path: /metrics
+          scheme: http
+          dns_sd_configs:
+          - names:
+            - kubecost-cost-analyzer.kubecost
+            type: 'A'
+            port: 9003
+        - job_name: kubecost-networking
           kubernetes_sd_configs:
-              - role: node
-          metric_relabel_configs:
-              - source_labels: [__name__]
-@
+            - role: pod
+          relabel_configs:
+          # Scrape only the the targets matching the following metadata
+            - source_labels: [__meta_kubernetes_pod_label_app]
+              action: keep
+              regex:  'kubecost-network-costs'
+  
+EOF
+(export NAMESPACE=kubecost && kubectl apply -n $NAMESPACE -f -)
+
+
+MANIFEST_URL=https://raw.githubusercontent.com/grafana/agent/v0.24.2/production/kubernetes/agent-bare.yaml NAMESPACE=kubecost /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/grafana/agent/v0.24.2/production/kubernetes/install-bare.sh)" | kubectl apply -f -
+
+```
+> Note: Verify if Grafana-agent is scraping data (optional)
+
+```Bash
+kubectl -n kubecost logs grafana-agent-0
 ```
 
-Next, apply the changes in the same namespace as your Grafana Agent deployment:
+To learn more about how to install and config Grafana agent as well as additional scrape configuration, please refer to [Grafana Agent for Kubernetes](https://grafana.com/docs/grafana-cloud/kubernetes/agent-k8s/k8s_agent_metrics/) section of the Grafana Cloud documentation. Or you can check Kubecost Prometheus scrape config at this [Github repository](https://github.com/kubecost/cost-analyzer-helm-chart/blob/ebe7e088debecd23f90e6dd75b425828901a246c/cost-analyzer/charts/prometheus/values.yaml#L1152)
 
-```sh
-$ kubectl apply extra_scrape_configs.yaml -n <namespace>
+## Step 2: Create dbsecret to allow Kubecots to query the metrics from Grafana Cloud Prometheus:
+
+- Create two files in your working directory, called `USERNAME` and `PASSWORD` respectively
```

Comment:
Hi Brett, thank you so much. It is intentional to be in all caps and without a file extension. It will work in Linux or *Nix like systems. Following my commands, customer can create those file as regular text files.